### DISCUSSION TITLE
WSJT-X and bandmap window

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [foldynl/QLog#446] [Ideas]
Hello!

Version 0.38.0 added this:
[NEW] - WSJTX tunes freq/mode like Rig if rig is disconnected
and it works great.
But would it be possible for the BANDMAP window to follow the frequency as well?

It would help a bit with callsign search.

**Comment by foldynl** [comment]
Thanks for reporting. It is a bug. I am opening an issue for it.
